swig/cpp/src/Libyang.cpp

```diff
@@ -29,107 +29,96 @@ extern "C" {
 #include "tree_schema.h"
 }
 
-using namespace std;
-
-Context::Context(ly_ctx *ctx, S_Deleter deleter) {
-    _ctx = ctx;
-    _deleter = deleter;
-}
+Context::Context(ly_ctx *ctx, S_Deleter deleter):
+    ctx(ctx),
+    deleter(deleter)
+{};
 Context::Context(const char *search_dir, int options) {
-    _ctx = ly_ctx_new(search_dir, options);
-    if (NULL == _ctx) {
```

Comment:
nit: if I were writing these, I would just do:

    if (!ctx) {
        ....

Reply:
I'm not sure about the code style myself, I started using `if (NULL == _ctx) {` so it was more consistent with other parts of the code but if you think it's better C++ practice, I can switch.

Reply:
done.